source/user-manual/api/rbac/reference.rst

```diff
@@ -482,6 +495,21 @@ mitre:read
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 - :api-ref:`GET /mitre <operation/api.controllers.mitre_controller.get_attack>` (`*:*`_)
 
+Rootcheck
```

Comment:
Also add agent:group (I know it is not present on the others, but a PR has been included to 4.0.1 to add this, so we need to add it to new additions)